### ISSUE TITLE
Unicode characters are not suggested for completion in the main editor

### ISSUE THREAD

**Issue by amrods** [julia-vscode/julia-vscode#1921]
For example `\alpha` no longer suggests `α` in the main editor. In fact, I think right now it is impossible to type `α` in the main editor.

**Comment by davidanthoff** [comment]
Hm, works just fine for me...

**Comment by amrods** [comment]
Should have said this: I'm using VSCode 1.53.1 and extension version 1.1.15, (and Julia 1.5.3).

Could just be me. If I remember correctly someone had the same problem for a previous release. I'll try to find out what happened then.

**Comment by amrods** [comment]
I couldn't find anything.

I am now finding that the issue arises randomly. I though perhaps it was because the Julia Language Server crashed initially, but now I'm not sure.

**Comment by dhananjayghei** [comment]
I am also experiencing similar problems on Ubuntu 20.04, VSCode 1.53, Julia version 1.5.3, and extension version 1.1.15.
1. Language server keeps crashing, 
2. Unicode completion does not work, and
3. Running the contributed command: 'language-julia.executeCodeBlockOrSelectionAndMove' failed.
For now, I have moved back to the extension version 1.0.10 and it seems to work fine.

**Comment by amrods** [comment]
Can confirm that moving back to version 1.0.10 works fine.

**Comment by ZacLN** [comment]
Is there anything listed under 'Julia Language Server' on the `Output` pane of vscode?

**Comment by pfitzseb** [comment]
Yeah, this is the typical symptom of the LS not starting properly.

**Comment by dhananjayghei** [comment]
The `Output` pane gives the same message as in #1919

**Comment by amrods** [comment]
This is what I get:
```
[ Info: Starting the Julia Language Server
[ Info: Symbol server store is at '/Users/miguel/Library/Application Support/VSCodium/User/globalStorage/julialang.language-julia/symbolstorev4'.
┌ Warning: Artifacts not stored on disc
└ @ SymbolServer ~/.vscode-oss/extensions/julialang.language-julia-1.1.16/scripts/packages/SymbolServer/src/SymbolServer.jl:197
┌ Warning: ZipFile not stored on disc
└ @ SymbolServer ~/.vscode-oss/extensions/julialang.language-julia-1.1.16/scripts/packages/SymbolServer/src/SymbolServer.jl:197
┌ Warning: Artifacts not stored on disc
└ @ SymbolServer ~/.vscode-oss/extensions/julialang.language-julia-1.1.16/scripts/packages/SymbolServer/src/SymbolServer.jl:197
[ Info: Received new data from Julia Symbol Server.
┌ Warning: ZipFile not stored on disc
└ @ SymbolServer ~/.vscode-oss/extensions/julialang.language-julia-1.1.16/scripts/packages/SymbolServer/src/SymbolServer.jl:197
[ Info: Received new data from Julia Symbol Server.
```

**Comment by lrsantos11** [comment]
I have the same issues. That is my system
```
Julia Version 1.6.1
Commit 6aaedecc44 (2021-04-23 05:59 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Xeon(R) W-2133 CPU @ 3.60GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-11.0.1 (ORCJIT, skylake-avx512)
```

**Comment by chelseas** [comment]
I'm having the same isue
```
julia> versioninfo()
Julia Version 1.6.1
Commit 6aaedecc44 (2021-04-23 05:59 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: AMD Ryzen 7 4800H with Radeon Graphics
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-11.0.1 (ORCJIT, znver2)
```

Julia extension for VScode v1.2.7

**Comment by skanskan** [comment]
I have the same problem.
I'm using:
Visual Code Studio v.1.62.0
Julia v1.6.3
Julia Extension v.1.4.3 (and the same for Julia Insider v.1.4.3)

Autocompletion for symbols such as α and ∘ works on the REPL but it doesn't on the main VS Code workplace.

Another issue
https://github.com/julia-vscode/julia-vscode/issues/613
suggests adding this to the settings.json file:

```
"[julia]": {
    "editor.tabCompletion": "on",
        "editor.quickSuggestions": true
}
```

But it doesn't work either.

Other extensions allows you to insert unicode and latex symbols, buy every time to want to insert a code you need to type ctrl+p go to Unicode: Insert Math symbol...

**Comment by skanskan** [comment]
I have uninstalled the Julia VS Code extension v.1.4.3 and installed the new 1.5.4 and now the symbol autocompletion seems to work.
Also had to delete the .julia folder and reinstall everything because it produced errors.

**Comment by LilithHafner** [comment]
Can reproduce on julia-vscode v1.6.14, and 
```
julia> versioninfo()
Julia Version 1.7.2
Commit bf53498635 (2022-02-06 15:21 UTC)
Platform Info:
  OS: macOS (x86_64-apple-darwin19.5.0)
  CPU: Intel(R) Core(TM) i5-8210Y CPU @ 1.60GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-12.0.1 (ORCJIT, skylake)
Environment:
  JULIA_EDITOR = code
  JULIA_NUM_THREADS = 
```
(sic)

**Comment by LilithHafner** [comment]
Workaround: copy and paste from the REPL.

**Comment by IvaroEkel** [comment]
Simple and unexpected solution: I just tried in the console \alpha and then tab, the greek letter appeared in the REPL. Then I did nothing but go to my script again and did the same and now unicode recognition works just fine.

**Comment by biona001** [comment]
I am experiencing the same issue on Julia 1.7.3, VS code 1.67.2, and Julia language support 1.6.17.

**Comment by goerz** [comment]
Would it be possible to somehow separate unicode entry from the LanguageServer? This feels like it should be a standalone feature. I've found https://marketplace.visualstudio.com/items?itemName=oijaz.unicode-latex, which seems to work, but not in Julia files. Maybe it would be better to remove the feature from the Julia plugin and recommend people install that plugin or a similar one instead?

**Comment by davidanthoff** [comment]
The language server backend was rewritten in v1.220.0: StaticLint.jl and SymbolServer.jl were folded into JuliaWorkspaces, and completion, hover, signature help, go-to-definition and find-references were reimplemented. This report predates that rewrite, so the code that produced it no longer exists and the behaviour cannot be confirmed against the current engine.

Closing as outdated. If you still see this on a current release, please open a fresh issue with a reproducer and we will look at it against the new engine.
